### DISCUSSION TITLE
¿Tareas y Practicas? - ¿Donde se suben o presentan?

### DISCUSSION THREAD

**Discussion by MarcoAntonio159** [Launch-X-Latam/MisionFrontEnd#314] [Q&A]
Buenas tardes con todos,

Recien empezare a realizar las tareas de cada modulo, por lo que requiero el apoyo de la comunidad para resolver unas cuantas dudas:

1. Ya se definio, donde se suben las tareas?
2. Me podrian listar los enlaces de las clases, donde se explique que se tiene que hacer en la tarea.

De antemano agradesco su apoyo!

Saludos cordiales.

**Comment by zaazilcb** [comment]
practica 1: (hasta donde se no se entrega)
https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/01%20-%20INTRO/practicas
practica 2: (4 noviembre)
https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML/practicas
practica 3: (esta mal la fecha pero quiza sea igual el 4)
https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/03%20-%20CSS/practica

Aun no dicen como se subirán las practicas pero lo mas probable es que tengas que mandar los links de tus repositorios, el 4 diran como se procederá con eso

**Reply by JoseBenin82** [reply]
Hola chicos el sherpa menciono que suban sus practicas a su repo...! lo dijo ayer en la sesion.

**Reply by MarcoAntonio159** [reply]
@

> Hola chicos el sherpa menciono que suban sus practicas a su repo...! lo dijo ayer en la sesion.

Compañero, por lo que entiendo no hay practica de JavaScript?

**Reply by safraZX** [reply]
hola si dice que el 4 se subira un form pero donde esta el form ? me estoy poniendo al dia hasta donde se solo debemos hacerlas y dejarlas en un repo todas
